src/_guides/libraries/writing-package-pages.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Writing package pages
+description: Learn how to write a good package page.
+---
+
+<style>
+  .screenshot, .screenshot-narrow {
+    width: 90%;
+    align-content: center;
+    border-style: solid;
+    border-width: 1px;
+    border-color: lightgray;
+    padding: 10px
+  }
+
+  .screenshot-narrow {
+    max-width: 400px;
+  }
+</style>
+
+This document describes what you can do to
+construct a good package page on pub.dev.
+More specifically, this document provides several tips on
+how to write a better README.md file,
+as marked in the following image.
```

Comment:
Can you redo this figure? Two reasons:

* It doesn't include the title **url_launcher** in the area of **This document**, but that text _is_ provided by the README.
* It shows the title **url_launcher**. 😆  We probably want to show a better example, no?

Could we use in_app_purchase, yaml, or another nicely done package page?

Reply:
I wonder if we should change "This document" to "README file (this document)"

Reply:
The label "this document" is updated. And I replaced the image with `shared_preferences` page, which has a title `Shared preferences plugin`. Would this work?

Otherwise, we'll have to update one of the package's readme and retry. All the 1P Flutter Favorite packages have some sort of titles in the body, ironically :P

Reply:
I'll take another look Monday